pkg/infra/block_collector_test.go

```diff
@@ -0,0 +1,50 @@
+package infra_test
+
+import (
+	"sync"
+	"tape/pkg/infra"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("BlockCollector", func() {
+
+	Context("Commit", func() {
+		It("should return true when observer is 1", func() {
+			instance := infra.NewBlockCollector(1, 1)
+			Expect(instance.Commit(uint64(1), "addr")).To(BeTrue())
+		})
+
+		It("Should added when the block committed with threshold at 1st time", func() {
```

Comment:
this sentence doesn't read right... and it's exactly the same as the test case below